### PR TITLE
feat(android): Add autoinstallation sections for okhttp, fragment and timber

### PR DESCRIPTION
**Pull request by romtsn**
Merge after 3.1.0 of SAGP is shipped

### CODE REVIEW

File: src/platforms/android/configuration/integrations/timber.mdx [hunk 1]
```diff
@@ -20,7 +20,47 @@ Starting from version `5.6.2`, the `Timber.tag` usage is no longer supported by
 
 </Note>
 
-## Install
+## Automatically with the Sentry Android Gradle plugin
```
Comment (imatwawana) [inline src/platforms/android/configuration/integrations/timber.mdx:23]:
```suggestion
## Auto-Installation With the Sentry Android Gradle Plugin
```

File: src/platforms/android/configuration/integrations/timber.mdx [hunk 2]
```diff
@@ -20,7 +20,47 @@ Starting from version `5.6.2`, the `Timber.tag` usage is no longer supported by
...
+
+### Install
+
+Starting from version `3.1.0`, the Sentry Android Gradle plugin will automatically add the `sentry-android-timber` dependency. The plugin will only add the `sentry-android-timber` dependency in case a `timber` dependency was discovered on the classpath.
```
Comment (imatwawana) [inline src/platforms/android/configuration/integrations/timber.mdx:27]:
```suggestion
Starting from version `3.1.0`, the Sentry Android Gradle plugin will automatically add the `sentry-android-timber` dependency. The plugin will only add the `sentry-android-timber` dependency if a `timber` dependency was discovered on the classpath.
```

File: src/platforms/android/configuration/integrations/timber.mdx [hunk 3]
```diff
@@ -20,7 +20,47 @@ Starting from version `5.6.2`, the `Timber.tag` usage is no longer supported by
...
+    mavenCentral()
+  }
+}
+
+plugins {
+  id("io.sentry.android.gradle") version "3.1.0-alpha.1"
+}
+```
+
+and initialize the [Android SDK](/platforms/android/).
```
Comment (imatwawana) [inline src/platforms/android/configuration/integrations/timber.mdx:55]:
```suggestion
Then, initialize the [Android SDK](/platforms/android/).
```

File: src/platforms/android/configuration/integrations/timber.mdx [hunk 4]
```diff
@@ -20,7 +20,47 @@ Starting from version `5.6.2`, the `Timber.tag` usage is no longer supported by
...
+
+The Android SDK automatically adds the `SentryTimberIntegration`, if the `sentry-android-timber` dependency was found on the classpath. The integration is added with `minEventLevel` set to `ERROR` and `minBreadcrumbLevel` set to `INFO`.
```
Comment (imatwawana) [inline src/platforms/android/configuration/integrations/timber.mdx:57]:
```suggestion
The Android SDK automatically adds the `SentryTimberIntegration` if the `sentry-android-timber` dependency was found on the classpath. The integration is added with `minEventLevel` set to `ERROR` and `minBreadcrumbLevel` set to `INFO`. 
```

File: src/platforms/android/configuration/integrations/timber.mdx [hunk 5]
```diff
@@ -20,7 +20,47 @@ Starting from version `5.6.2`, the `Timber.tag` usage is no longer supported by
...
+}
+```
+
+and initialize the [Android SDK](/platforms/android/).
+
+The Android SDK automatically adds the `SentryTimberIntegration`, if the `sentry-android-timber` dependency was found on the classpath. The integration is added with `minEventLevel` set to `ERROR` and `minBreadcrumbLevel` set to `INFO`. 
+
+However, you can still override the default behaviour by adding your own instance of the `SentryTimberIntegration`. For that, refer to the [manual installation](/platforms/android/configuration/integrations/timber/#configure) section below.
+
+## Manually
```
Comment (imatwawana) [inline src/platforms/android/configuration/integrations/timber.mdx:61]:
```suggestion
## Manual Installation
```

**Review by imatwawana — COMMENTED** [review]
Made some wording and style fixes

File: src/platforms/android/configuration/integrations/fragment.mdx [hunk 6]
```diff
@@ -12,7 +12,47 @@ The `sentry-android-fragment` library provides [Fragment](https://developer.andr
...
+
+```groovy
+buildscript {
+  repositories {
+    mavenCentral()
+  }
+}
+
+plugins {
+  id "io.sentry.android.gradle" version "3.1.0-alpha.1"
```
Comment (bruno-garcia) [inline src/platforms/android/configuration/integrations/fragment.mdx:31]: Should put out an RC before merging this?

**Review by bruno-garcia — APPROVED** [review]

File: src/platforms/android/configuration/integrations/fragment.mdx [hunk 7]
```diff
@@ -12,7 +12,47 @@ The `sentry-android-fragment` library provides [Fragment](https://developer.andr
...
+plugins {
+  id("io.sentry.android.gradle") version "3.1.0-alpha.1"
+}
+```
+
+Then, initialize the [Android SDK](/platforms/android/).
+
+The Android SDK automatically adds the `FragmentLifecycleIntegration` if the `sentry-android-fragment` dependency was found on the classpath. The integration is added with both `enableFragmentLifecycleBreadcrumbs` and `enableAutoFragmentLifecycleTracing` enabled. 
+
+However, you can still override the default behaviour by adding your own instance of the `FragmetnLifecycleIntegration`. For that, refer to the [manual installation](/platforms/android/configuration/integrations/fragment/#configure) section below.
```
Comment (romtsn) [inline src/platforms/android/configuration/integrations/fragment.mdx:51]:
```suggestion
However, you can still override the default behaviour by adding your own instance of the `FragmentLifecycleIntegration`. For that, refer to the [manual installation](/platforms/android/configuration/integrations/fragment/#configure) section below.
```

File: src/platforms/android/configuration/integrations/fragment.mdx [hunk 8]
```diff
@@ -12,7 +12,47 @@ The `sentry-android-fragment` library provides [Fragment](https://developer.andr
...
+    mavenCentral()
+  }
+}
+
+plugins {
+  id("io.sentry.android.gradle") version "3.1.0-alpha.1"
+}
+```
+
+Then, initialize the [Android SDK](/platforms/android/).
```
Comment (romtsn) [inline src/platforms/android/configuration/integrations/fragment.mdx:47]:
```suggestion
Then, initialize the [Android SDK](/platforms/android/#configure).
```

File: src/platforms/android/configuration/integrations/okhttp.mdx [hunk 9]
```diff
@@ -20,7 +20,76 @@ The minimum supported version of `okhttp` is `3.13.0` due to its [incompatibilit
...
+    mavenCentral()
+  }
+}
+
+plugins {
+  id("io.sentry.android.gradle") version "3.1.0-alpha.1"
+}
+```
+
+Then, initialize the [Android SDK](/platforms/android/).
```
Comment (romtsn) [inline src/platforms/android/configuration/integrations/okhttp.mdx:55]:
```suggestion
Then, initialize the [Android SDK](/platforms/android/#configure).
```

File: src/platforms/android/configuration/integrations/timber.mdx [hunk 10]
```diff
@@ -20,7 +20,47 @@ Starting from version `5.6.2`, the `Timber.tag` usage is no longer supported by
...
+    mavenCentral()
+  }
+}
+
+plugins {
+  id("io.sentry.android.gradle") version "3.1.0-alpha.1"
+}
+```
+
+Then, initialize the [Android SDK](/platforms/android/).
```
Comment (romtsn) [inline src/platforms/android/configuration/integrations/timber.mdx:55]:
```suggestion
Then, initialize the [Android SDK](/platforms/android/#configure).
```

**Pull request merged**